packages/react-percy-config/src/getWebpackSettings.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-console */
+
+import chalk from 'chalk';
+import path from 'path';
+
+class ConfigError extends Error {
+  constructor(message) {
+    super(message);
+
+    Error.stackTraceLimit = 0;
+    Error.captureStackTrace(this, ConfigError);
+
+    this.name = '';
+    this.message = chalk.bold.red(`Error: ${message}`);
+  }
+}
+
+function sanitizeCustomWebpackConfig(config = {}) {
+  if (typeof config !== 'object') {
+    throw new ConfigError(
+      `The \`webpack\` field in percy.config.js must be a plain object, but was "${typeof config}".`,
+    );
+  }
+
+  if (Array.isArray(config)) {
+    throw new ConfigError(
+      'The `webpack` field in percy.config.js must be a plain object, but was "array".',
+    );
+  }
+
+  // Do not allow users to customize the entry config
+  if (config.entry) {
+    console.log(chalk.bold.yellow('Warning: ignoring `webpack.entry` field in percy.config.js.'));
+    delete config.entry;
+  }
+
+  // Do not allow users to customize the output config
+  if (config.output) {
+    console.log(chalk.bold.yellow('Warning: ignoring `webpack.output` field in percy.config.js.'));
+    delete config.output;
+  }
+
+  // Move module.loaders to module.rules so we can properly merge with them
+  if (config.module) {
+    if (config.module.loaders) {
+      console.log(
+        chalk.bold.yellow(
+          'Warning: found `webpack.module.loaders` field in percy.config.js, but react-percy requires a webpack 3 compatible config.',
```

Comment:
When I read the first line here, before reaching the second line, I thought "oh oh, error, this is going break", but on the next line I find out you've handled it.

Maybe a tiny improvement would be to reorder the wording:
```
Warning: found `webpack.module.loaders` field in percy.config.js, and moved them to  `webpack.module.rules`.
webpack.module.loaders was moved to meet react-percy's requirement of a webpack 3 compatible config.
```

Reply:
Updated:

![image](https://user-images.githubusercontent.com/733696/30515644-2767886a-9afa-11e7-8886-767a49b58fe8.png)